From 5c9bf430dfa9ad75fedf342a4a2be88fa31cd309 Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Thu, 17 Mar 2022 12:37:57 +0100
Subject: [PATCH] Revert "kernel-install: prefer /boot over /boot/efi for
 $BOOT_ROOT"

This reverts commit d0e98b7a1211412dccfcf4dcd2cc0772ac70b304.

Related: #2065061

[msekleta: this revert is done in order to make backporting easier,
patch will be reapplied later.]
---
 src/kernel-install/kernel-install | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/kernel-install/kernel-install b/src/kernel-install/kernel-install
index d85852532b..b358b03b2f 100755
--- a/src/kernel-install/kernel-install
+++ b/src/kernel-install/kernel-install
@@ -108,7 +108,7 @@ fi
 [ -z "$MACHINE_ID" ] && MACHINE_ID="Default"
 
 [ -z "$BOOT_ROOT" ] && for suff in "$MACHINE_ID" "loader/entries"; do
-    for pref in "/efi" "/boot" "/boot/efi" ; do
+    for pref in "/efi" "/boot/efi" "/boot"; do
         if [ -d "$pref/$suff" ]; then
             BOOT_ROOT="$pref"
             break 2
